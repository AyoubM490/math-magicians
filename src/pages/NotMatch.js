import React from 'react';

const NotMatch = () => {
  const style = {
    color: 'red',
    fontSize: '25px',
    textAlign: 'center',
    marginTop: '5rem',
  };
  return (
    <div>
      <div>
        <h1 style={style}>There is nothing on this page</h1>
      </div>
    </div>
  );
};

export default NotMatch;
