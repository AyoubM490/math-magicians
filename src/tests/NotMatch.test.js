import React from 'react';
import renderer from 'react-test-renderer';
import NotMatch from '../pages/NotMatch';

describe('test if NotMatch', () => {
  test('renders correctly', () => {
    const component = renderer
      .create(<NotMatch />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
