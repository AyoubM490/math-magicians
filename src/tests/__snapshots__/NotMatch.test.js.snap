// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test if NotMatch renders correctly 1`] = `
<div>
  <div>
    <h1
      style={
        Object {
          "color": "red",
          "fontSize": "25px",
          "marginTop": "5rem",
          "textAlign": "center",
        }
      }
    >
      There is nothing on this page
    </h1>
  </div>
</div>
`;
